modules/dop/endpoints/pipeline.go

```diff
@@ -434,7 +441,7 @@ func (e *Endpoints) UpdateCmsNsConfigs(userID string, orgID uint64) error {
 			PipelineSource: apistructs.PipelineSourceDice.String(),
 			KVs: map[string]*cmspb.PipelineCmsConfigValue{
 				utils.MakeOrgGittarUsernamePipelineCmsNsConfig(): {Value: "git", EncryptInDB: true},
-				utils.MakeOrgGittarTokenPipelineCmsNsConfig():    {Value: members[0].Token, EncryptInDB: true}},
+				utils.MakeOrgGittarTokenPipelineCmsNsConfig():    {Value: res.Data[0].AccessKey, EncryptInDB: true}},
```

Comment:
mark a TODO comment, we will use another way to help pipelines get good token, but that is not PAT.

Reply:
done